types/src/keyless/bn254_circom.rs

```diff
@@ -240,7 +243,10 @@ pub fn get_public_inputs_hash(
 ) -> anyhow::Result<Fr> {
     if let EphemeralCertificate::ZeroKnowledgeSig(proof) = &sig.cert {
         let (has_extra_field, extra_field_hash) = match &proof.extra_field {
-            None => (Fr::zero(), Fr::zero()),
+            None => (
+                Fr::zero(),
+                // When extra_field is none, set the hash to the constant below which is equal to the hash of a single space string.
+                Fr::from_le_bytes_mod_order(&BigUint::from_str("8441624714246672702469557419929302030072955765775944993548127422079350103481")?.to_bytes_le())),
```

Comment:
Can you define this hash as a `Lazy<Fr>` and reference it here? Hardcoded hashes are the worst.

Reply:
+1